lib/killbill_client/models/transaction.rb

```diff
@@ -138,6 +138,59 @@ def cancel_scheduled_payment(user = nil, reason = nil, comment = nil, options =
                           }.merge(options)
       end
 
+      def chargerback_by_external_key(payment_external_key, amount, currency, effective_date = nil, user = nil, reason = nil, comment = nil, options = {}, refresh_options = nil)
```

Comment:
It's _chargeback_, not charge**r**back (https://en.wikipedia.org/wiki/Chargeback).